app/models/candidate.rb

```diff
@@ -16,10 +16,24 @@ class Candidate < ApplicationRecord
   belongs_to :course_from_find, class_name: 'Course', optional: true
   belongs_to :fraud_match, optional: true
 
+  PUBLISHED_FIELDS = %w[email_address].freeze
+
   after_create do
     update!(candidate_api_updated_at: Time.zone.now)
   end
 
+  before_save do |candidate|
+    if (candidate.changed & PUBLISHED_FIELDS).any?
+      touch_application_choices
+    end
+  end
+
+  def touch_application_choices
+    return unless application_choices.any?
+
+    application_choices.touch_all
```

Comment:
should we scope to current year? I think we should but how does that affect old candidates?